compiler/lib/src/main/scala/codegen/CppWriter/ComputeCppFiles.scala

```diff
@@ -105,9 +105,35 @@ object ComputeCppFiles {
     /** Gets the C++ file name for generated component test harness helpers */
     def getComponentTestHelper(baseName: String) = s"${baseName}TesterHelpers"
 
+    /* Checks whether the file name is a tester helpers file */
+    def isTesterHelpers(fileName: String): Boolean =
+      fileName.matches(".*TesterHelpers\\.cpp")
+
     /** Gets the C++ file name for generated component main test harness classes */
     def getComponentTestMain(baseName: String) = s"${baseName}TestMain"
 
+    /* Maps generated template patterns to handcoded file patterns */
+    val templateMap = Map(
+      "TestMain\\.cpp$" -> "TestMain.cpp",
+      "Tester\\.cpp$" -> "Tester.cpp",
+      "Tester\\.hpp$" -> "Tester.hpp",
```

Comment:
Are these mapping to the same thing?

Reply:
Yes, the map is the identity map in those cases.